pay-stamp-duty-land-tax/_title.html

```diff
@@ -1,18 +1,20 @@
-<div class="grid-row">
+<div class="grid-row sidebar-with-body">
   <div class="column-two-thirds">
-    <div class="govuk-title length-long">
+    <div class="govuk-title length-average">
```

Comment:
Any reason why this and the sidebar-with-body classes are different on this one but not the thresholds one? not sure which one is correct.